### ISSUE TITLE
Error in GlobalDataMiddleware when English locale is removed

### ISSUE THREAD

**Issue by istride** [unicef/iogt#1397]
When the 'English' locale is removed the following error occurs when navigating to the admin UI (/admin/):

[locale_not_exist_error.log](https://github.com/unicef/iogt/files/9507066/locale_not_exist_error.log)

Presumably, the locale was deleted because it is deemed not to be required by the site owners, and/or to remove the 'English' option from the language switcher on the site.

**Comment by istride** [comment]
Is the English locale required for the Wagtail Admin to function?

If not, then perhaps the middleware needs to be changed to handle locales differently.

If so, then perhaps some work needs to be done on the language switcher to be able to exclude certain locales.

**Comment by cbunicef** [comment]
Yes the admins deleted English locale because they didn't need it.

We do have the ability to hide a locale from the language switcher using Settings > Locale Details, by creating a Detail for English and deselecting "Active". I explained that to the Admin that this is a safer option. I did realize though that this doesn't prevent users from accessing the URLs for that locale - which is a good reason for an Admin to delete a locale entirely.
